crates/tabby-common/src/index.rs

```diff
@@ -101,6 +98,13 @@ impl CodeSearchSchema {
                 .map(|x| Term::from_field_text(self.field_body, x)),
         ))
     }
+
+    pub fn git_url_query(&self, git_url: &str) -> Box<TermQuery> {
```

Comment:
add unit test for coverage

Reply:
Not sure what there is to test with this function?